CONTRIBUTING.md

```diff
@@ -21,6 +22,44 @@ git remote add upstream https://github.com/FerretDB/FerretDB.git
 
 ### Setting up the development environment
 
+In the github-actions there are checks that new code complies the imports sorted convention
+and that the code is gofmt-ed. To avoid failing PR checks, please setup your beloved text editor
+to gofmt files before push. Here provided some setup guides for VScode and Goland.
```

Comment:
Might it be easier to just recommend `task fmt lint`?

Reply:
I suggest giving several solutions that fits different people. 
With manual runs the risk of human factor "forgot to run" is higher.

Reply:
Also added `task fmt lint` to the guideline)